### ISSUE TITLE
Many to Many - Nuxt SSR - Pivot not updating

### ISSUE THREAD

**Issue by cmartinho** [vuex-orm/vuex-orm#312]
First of all, thank you for the awesome work and the time you put into this @kiaking.

Now, i've been having an issue with Nuxt (2.0.0) and Vuex-orm (0.31.6) pivot Base State.

I've got a many to many relationship declared and i'm using AsyncData to get the payload and then insert them into the models. 
When running it from the SSR, the **Pivot table does not get filled** (but the blocks / entry models get their data). 
Although, after navigating to another route or updating something on the code with hot-reload, the pivot table gets updated as expected. 
Here's the declaration of the relationships & an example of the payload:

Entry.js:

```
...
static fields() {
    return {
      id: this.attr(null),
      blocks: this.belongsToMany(Block, EntryBlock, "entry_id", "block_id")
    }
  }
...
```

Block.js
```
...
static fields() {
    return {
      id: this.attr(null)
    }
  }
...
```

EntryBlock.js

```
static primaryKey = ["block_id", "entry_id"]

  static fields() {
    return {
      block_id: this.attr(null),
      entry_id: this.attr(null)
    }
  }
```

Payload (Entry Example):
`
{
  "id": 1,
  "blocks": [
    {
      "id": 79,
    },
    {
      "id":    222,
    }
  ]
}
`

Insertion Method:
```
await app.$axios.$get(`/api/v1/entry${pageSlug}`).then(res => {
        data = res
      })
```

`await Entry.insertOrUpdate({ data })`

**Comment by dousybox** [comment]
You payload should like below:
(Entry Example):
```
{
  "id": 1,
  "blocks": [
    { "block_id": 79, "entry_id": 1 },
    { "block_id": 222, "entry_id": 1 }
  ]
}
```

**Comment by jpenaroche** [comment]
@dousybox  Well, i have this Many To Many Polimorphic Relation, and i pass the payload with the same fieldnames as declared on vuex-orm models..., but vue-orm doesn't insert these fields (tag, order). But the weird thing is that i set primaryKey attribute and then those fields are inserted but the query builder doesn't load.
My Other question is, when i set my primaryKey array vuex-orm duplicates each entry on store, one with my primaryKey setted, and the other one without my keys, and that's one the lib returns :\, why this happen...

`export class Media extends Model {
    static entity = 'medias'

    static fields() {
        return {
            id: this.attr(null),
            disk: this.string(''),
            directory: this.string(''),
            filename: this.string(''),
            extension: this.string(''),
            mime_type: this.string(''),
            attributes: this.string(''),
            aggregate_type: this.string(''),
            size: this.number(0),
            created_at: this.attr(null),
            updated_at: this.attr(null)    
        }
    }
}`

`export class Mediable extends Model {
    static entity = 'mediables'

    static primaryKey = ['media_id','mediable_id','mediable_type']

    static fields() {
        return {
            media_id: this.attr(null),
            mediable_id: this.attr(null),
            mediable_type: this.attr(null),
            tag: this.attr(null),
            order: this.attr(null),
        }
    }
}`

`export class User extends Model {
    static entity = 'users'

    static fields() {
        return {
            id: this.attr(null),
            name: this.string(''),
            email: this.string(''),
            active: this.boolean(true),
            is_admin: this.boolean(false),
            locale: this.string('es'),
            created_at: this.attr(''),
            updated_at: this.attr(''),
            remember_token: this.string(''),         
            media: this.morphToMany(Media, Mediable, 'media_id', 'mediable_id', 'mediable_type')         
        }
    }
}`

These are my store after fetching data and my data retrieved by query

![Screenshot at 2019-04-04 10-03-57](https://user-images.githubusercontent.com/17749214/55562177-6da94f00-56c1-11e9-8445-e5ec7ad47e1b.png)
![Screenshot at 2019-04-04 10-04-21](https://user-images.githubusercontent.com/17749214/55562178-6da94f00-56c1-11e9-9490-08e8c3ac2f2a.png)

**Comment by narnikgamarnikus** [comment]
The same issue

**Comment by kiaking** [comment]
Hi! Does anybody have update on this issue?

**Comment by kiaking** [comment]
I'll be closing this issue due to inactivity. Please feel free to open another issue if there is anything! 🤝
